package.json

```diff
@@ -94,7 +95,6 @@
     "eslint-plugin-markdown": "^2.0.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^5.1.0",
-    "eslint-plugin-standard": "^5.0.0",
```

Comment:
If this is removed, what can make the code follows standard style?

Reply:
According to the deprecation message [here](https://www.npmjs.com/package/eslint-plugin-standard),

> eslint-config-standard 16.0.0 no longer require the eslint-plugin-standard package